fxmanifest.lua

```diff
@@ -9,11 +9,13 @@ ox_lib 'locale'
 
 shared_scripts {
 	'@ox_lib/init.lua',
-	'@qbx_core/modules/utils.lua',
 }
 
 server_script 'server/main.lua'
-client_script 'client/main.lua'
+client_scripts {
+    '@qbx_core/modules/lib.lua',
+    'client/main.lua'
+}

```

Comment:
Lib goes in shared but that's just me

Reply:
It goes wherever it's used, no need to put it in shared if we only use it client side 🤔

Reply:
sounds french but okay